### ISSUE TITLE
Feature request: Show docset size before download 

### ISSUE THREAD

**Issue by mandeldl** [Kapeli/Dash-iOS#57]
On iOS devices it'd be helpful to know in advance how large a docset is.

**Comment by nexon** [comment]
@Kapeli Can we add a node or attribute to the xml of the each feed that give you the size?. After that it will be easy to add the file size to the app.

**Comment by Kapeli** [comment]
@nexon Have you considered sending a HEAD request to the server requesting the archive and checking the Content-Length?

**Comment by nexon** [comment]
@Kapeli Yes, but is not the best option here since you will need to send N request where N is the quantity of visible cells, in which case is a waste of resources.

Also you need to consider cancelling the request when the user press for downloading. 
Another problem is what happen with the request (HEAD) when the user scroll. Should be cancelled?, we should get a reference to the Feed Object when doing the request so we don't cancel it and we wait for the result and we put it in the Feed Object (we need to avoid leaking memory)?

**Comment by Kapeli** [comment]
How would all this be different if you had an attribute in the XML itself?

**Comment by stevapple** [comment]
For main docsets and cheat sheets, maybe you can maintain an XML feed especially for the sizes of each docset and cheatsheet? For 3rd-party ones, maybe sizes can be integrated into the main feed.

**Comment by Kapeli** [comment]
For main docsets I plan on moving to a JSON feed (like the cheatsheet one), which will include sizes. This won't come anytime soon though.

For cheatsheets I don't see a point in showing the size. All of them are less than 200 KB in size. Only `LaTeX_Math_Symbols`  is 1.1MB.
